Source/Scene/ImplicitTileCoordinates.js

```diff
@@ -35,12 +37,42 @@ import ImplicitSubdivisionScheme from "./ImplicitSubdivisionScheme.js";
 export default function ImplicitTileCoordinates(options) {
```

Comment:
Also maybe include some notes about the limitations on the range level, x, y, z imposed by the bit twiddling and `MortonOrder`?